assemble/src/main/scripts/accumulo-client-build.jsh

```diff
@@ -0,0 +1,53 @@
+/*
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+*/
+  String accumuloProp = 
+      System.getProperty("user.dir").replace("/bin", "/conf/accumulo-client.properties");
+  String name; String zk; String principal; String token; 
+  AccumuloClient client = null; Authorizations auth = null;
+  
+  try {
+    // Does the accumulo properties file exists?
+    if (Files.exists(Paths.get(accumuloProp))) {
+      // Build Accumulo Client
+      client = Accumulo.newClient().from(accumuloProp).build();
+      
+      // Test Accumulo Client
+      auth = client.securityOperations().getUserAuthorizations(client.whoami());
+      if (auth != null) {
+        System.out.println("Accumulo Client was Successfully Built!");
+      System.out.println("Use " + "\"client\"" + " to interact with Accumulo \n");
```

Comment:
I'm not sure we need this. This will perform an operation on the cluster. I think we can just construct the client from the properties file, without performing any cluster operations. The user can validate the client themselves if they want to.

Reply:
@ctubbsii My initial thought was to test out the client build first so that the user will always know whether or not their accumulo-client.properties is properly configured. That way the user can fix the properties file right away before Accumulo tasks are executed. However I realized now that it won't make much of a difference whether the user fix the properties file issue before or after executing an Accumulo task. Either way an exception would be thrown and the user won't be able to execute the Accumulo task without fixing the properties file first. Therefore we can omit the testing portion.